INSTALL

```diff
@@ -36,6 +36,7 @@ Contents
               python-4suite-xml python-simplejson python-xml \
               gnuplot poppler-utils \
               gs-common clisp gettext libapache2-mod-wsgi unzip \
+              pkg-config \
```

Comment:
Hi @tiborsimko 

Thanks for the response. I found that without this line, pip didn't work - is there a corresponding pip package that should be called ?
Also, forgot to add : 

Distributor ID:	Ubuntu
Description:	Ubuntu 14.10
Release:	14.10
Codename:	utopic
(perhaps the newer ubuntu is causing problems?)